### ISSUE TITLE
bug: Mismatching session id when opts.sessionID is >= 128

### ISSUE THREAD

**Issue by OzairP** [PassTheMayo/minecraft-server-util#30]
**Describe the bug:**
There is nominal behavior for `0 < opts.sessionID <= 127`. When this session id is >=128 the session id in the packet is `239` instead of `128`

This raises the exception `Error: Session ID in response did not match client session ID`

The packet diagram suggests that `sessionID` is a 32bit integer.

**Code:**
```javascript
// Add logging to queryFull.js#L82

await minecraft.queryFull('127.0.0.1', { sessionID: 127 }); // sessionID: 127, opts.sessionID: 127

await minecraft.queryFull('127.0.0.1', { sessionID: 128 }); // sessionID: 239, opts.sessionID: 128
```

**Expected behavior:**
Matching session id and correct output

**Additional context**
Minecraft 1.16.4 sever running Spigot

**Comment by PassTheMayo** [comment]
I was able to reproduce this issue with the latest commit. This issue has to do with Minecraft only accepting the last 4 bits of each byte in the session ID, documented [here](https://wiki.vg/Query#Generating_a_Session_ID). I just published version `3.4.2` which should fix this issue entirely. Please update to `3.4.2` and re-verify that the issue has been fixed.

**Comment by OzairP** [comment]
Yep looks fixed for me. Thank you!
